packages/esm-patient-common-lib/src/workspaces/useWorkspaceWindowSize.tsx

```diff
@@ -1,65 +0,0 @@
-import React, { createContext, useContext, useMemo, useEffect, useCallback } from 'react';
-import { useWorkspaces } from './useWorkspaces';
-import { WorkspaceWindowState } from '@openmrs/esm-patient-common-lib';
-
-interface WorkspaceWindowSize {
-  size: WorkspaceWindowState;
-}
-
-interface WorkspaceWindowSizeProviderProps {
-  children?: React.ReactNode;
-}
-
-interface WorkspaceWindowSizeContext {
-  windowSize: WorkspaceWindowSize;
-  updateWindowSize?(value: WorkspaceWindowState): any;
-  active: boolean;
-}
-
-function reducer(state: WorkspaceWindowSize, action: WorkspaceWindowState): WorkspaceWindowSize {
-  switch (action) {
-    case 'minimized':
-    case 'reopened':
```

Comment:
Can you remove `reopened` from the WorkspaceWindowState type?

Reply:
Done